src/emulator/init.cpp

```diff
@@ -62,10 +62,14 @@ static void before_callback(const glbinding::FunctionCall &fn) {
 static void after_callback(const glbinding::FunctionCall &fn) {
     MICROPROFILE_LEAVE();
     for (GLenum error = glGetError(); error != GL_NO_ERROR; error = glGetError()) {
+#ifdef _DEBUG
```

Comment:
split the change to this file to a separate commit, it has nothing to do with the script change

Reply:
Done